src/lib.rs

```diff
@@ -19,7 +19,7 @@
  */
 #![doc = include_str!("../README.md")]
 
-mod crypto;
+pub mod crypto;
```

Comment:
Why make all of the content of crypto public? The user should need only the errors, right?

Reply:
done